<?xml version="1.0"?>
<launch>

    <include file="$(find smartsix_moveit_config)/launch/demo_gazebo.launch" /> 
    <node pkg="timed_roslaunch" type="timed_roslaunch.sh" args="7 smartsix_controller smartsix_controller.launch" name="timed_roslaunch" output="screen" />
    <node pkg="rqt_multiplot" type="rqt_multiplot" name="rqt_multiplot" />

</launch>